_posts/2016-xx-xx-release-0.16.md

```diff
@@ -119,6 +119,116 @@ If you are using the low-level qcprot algorithm your self intead of our provided
 wrappers you have to change your code since the API has changed. For more see
 the [CHANGELOG].
 
+## MemoryReader: Reading trajectories from memory
+
+MDAnalysis typically reads trajectories from files on-demand, so that it can efficiently deal with large trajectories - even those that do not fit in memory. However, in some cases, both for convenience and for efficiency, it can be an advantage to work with trajectories directly in memory. In this release, we have introduced a MemoryReader, which makes this possible.
+
+The MemoryReader works with numpy arrays, using the same format as that used by for instance `DCDReader.timeseries()`. You can create a Universe directly from such an array:
+
+```python
+from MDAnalysis import Universe
+from MDAnalysisTests.datafiles import DCD, PDB_small
+from MDAnalysis.coordinates.memory import MemoryReader
+
+# Create a Universe using a DCD reader
+universe = Universe(PDB_small, DCD)
+
+# Extract coordinates
+coordinates = universe.trajectory.timeseries()
```

Comment:
Good point to show case this here. I haven't thought about that.  But your example is only a explicit version of  `transfer_to_memory`, maybe we can have another example that can't be replaced with a `transfer_to_memory`?

The only one I can think of is creating a random trajectory.
```python
coordinates = np.random.uniform(size=(100, u.atoms.n_atoms, 3)).cumsum(0)
universe = Universe(PDB_small, coordinates, format=MemoryReader)
```